apis/management.cattle.io/v3/logging_types.go

```diff
@@ -105,6 +105,10 @@ type EmbeddedConfig struct {
 	DateFormat            string `json:"dateFormat,omitempty" norman:"required,type=enum,options=YYYY-MM-DD|YYYY-MM|YYYY,default=YYYY-MM-DD"`
 	ElasticsearchEndpoint string `json:"elasticsearchEndpoint,omitempty" norman:"nocreate"`
 	KibanaEndpoint        string `json:"kibanaEndpoint,omitempty" norman:"nocreate"`
+	RequestsMemery        int    `json:"requestsMemory,omitempty" norman:"default=500,min=500"`
+	RequestsCPU           int    `json:"requestsCPU,omitempty" norman:"default=1000,min=1000"`
```

Comment:
we should not cap in json tags; caps should be there only in golang vars. It shoudl be just requestCpu in json (you can check other *cpu vars under types, it follows this paradigm)